configure.ac

```diff
@@ -110,8 +110,14 @@ GEANY_CHECK_THE_FORCE dnl hehe
 # i18n
 GEANY_I18N
 
-GEANY_DATA_DIR=`eval echo ${datarootdir}/geany`
-AC_SUBST([GEANY_DATA_DIR])
+if test -z $is_win32; then
+  GEANY_DATA_DIR=`eval echo ${datarootdir}/geany`
+  AC_SUBST([GEANY_DATA_DIR])
+else
+  GEANY_DATA_DIR=`eval echo ${prefix}/data`
+  AC_SUBST([GEANY_DATA_DIR])
+  AC_SUBST([pkgdatadir],'${prefix}/data')
+fi
```

Comment:
Without questioning whether setting pkgdatadir like that is a good idea (I(m actually not sure), you could use `AM_COND_IF` instead of adding a new variable to `GEANY_CHECK_MINGW`.  Also, no need to `AC_SUBST` in each path, could be in the common one.

```
AM_COND_IF([MINGW],
		   [GEANY_DATA_DIR=`eval echo ${prefix}/data`
		    AC_SUBST([pkgdatadir],'${prefix}/data')],
		   [GEANY_DATA_DIR=`eval echo ${datarootdir}/geany`])
AC_SUBST([GEANY_DATA_DIR])
```